### ISSUE TITLE
Granted 0.3.0 - invalid syntax

### ISSUE THREAD

**Issue by pjhavariotis** [common-fate/granted#267]
I have just updated Granted to 0.3.0 and unfortunately, I cannot open a console to my existing AWS accounts.
My setup is Windows 10, with Firefox (latest version).
Below it the error I'm getting for one of my AWS accounts:
```
Opening a console for Production-Services in your browser...
parsing uid (S-1-5-21-273161372-3525547952-1395639491-311579): strconv.ParseUint: parsing "S-1-5-21-273161372-3525547952-1395639491-311579": invalid syntax
```

**Comment by pjhavariotis** [comment]
I reverted back the update by installing the previous version 0.2.9.
The problem is gone!

**Comment by chrnorm** [comment]
Thanks for the bug report @pjhavariotis! We dropped the [github.com/pkg/browser](https://github.com/pkg/browser) dependency as part of v0.3.0 which looks to have caused a couple of issues. We'll release a patch to fix these in Windows.

**Comment by chrnorm** [comment]
@pjhavariotis we have released [v0.3.1](https://github.com/common-fate/granted/releases/tag/v0.3.1) which fixes this issue. Let us know if you experience any other issues!

**Comment by pjhavariotis** [comment]
I have just tested it. Issue Fixed!
